CONTRIBUTING.md

```diff
@@ -53,6 +53,71 @@ A team member will review the pull request as soon as possible.
 
 Please assign yourself to a maximum of 1 issue at a time.
 
+## Adding a new game
+
+<details>
+<summary>To add a new game, you need to follow these steps (<em>click to expand</em>)</summary>
+
+1. Create a new subfolder for your game within the `src` folder. In this example we will call it `chess`
+2. Create an `index.html` file containing this template code:
+
+   ```html
+   <!DOCTYPE html>
+   <html lang="en">
+     <head>
+       <meta charset="UTF-8" />
+       <title>chess</title>
+
+       <link
+         rel="stylesheet"
+         href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap"
+       />
+       <link rel="stylesheet" href="../brain-gain/main.css" />
+       <link rel="icon" type="image/x-icon" href="../images/logo-favicon.png" />
+     </head>
+
+     <body>
+       <core-navbar>chess</core-navbar>
+
+       <core-modal>
+         <div class="modal-header">HOW TO PLAY CHESS</div>
+         <p class="modal-text">This is the help modal</p>
+       </core-modal>
+
+       Hello world!
+       <script type="module" src="../components/script/main.js"></script>
+     </body>
+   </html>
+   ```
+
+3. Edit the home page to add a link to your new game, by adding the following code to `src/index.html`:
+   ```html
+   <a href="./chess/index.html">Visit the chess game!</a>
```

Comment:
Consider including a note about updating the tests in `cypress/e2e/homepage.cy.js` as visiting the mini-games from the homepage are tested there. If you do, don't forget to add `id="chess-btn"` here. 

A note about how they will need to add an image, a description for the game and apply the same animations would also be good.